### DISCUSSION TITLE
iOS: Live activities

### DISCUSSION THREAD

**Discussion by cth-latest** [CommunityToolkit/Maui#905] [New Feature Discussions]
As you may know, Apple released recently the iOS 16.1 OS and that allow developers to create rich user experience.

Flutter already has it's own package implementation for that https://pub.dev/packages/live_activities .

It is planned to be implemented actually ?

**Comment by VladislavAntonyuk** [comment]
I can investigate it. If this framework is ported by Xamarin iOS team, then we can implement it

**Comment by VladislavAntonyuk** [comment]
But I can add, it is a very specific for iOS, so most likely it won't be a part of this library

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
